src/main/resources/flows-config.yaml

```diff
@@ -82,6 +82,7 @@ flow:
     nextScreens:
       - name: expeditedSnapMoneyOnHand
   expeditedSnapMoneyOnHand:
+    onPostAction: MaybeClearMoneyOnHandAmount
     beforeSaveAction: CheckExpeditedSnapEligibility
```

Comment:
I added a new action to clear out the money amount in the event that they said they had money, but then went back and changed their mind on that.